### ISSUE TITLE
What's "Expected LiFetime"?

### ISSUE THREAD

**Issue by jgarces02** [GuanLab/GuanRank-R#1]
Hi @Hongyang449, 

I've seen the paper where you describe this methodology and I found very interesting. I'm trying to use your code but it's a bit difficult without any documentation... so, I'd like ask you two little things:
- Is there any documentation or paper that helps me to understand a bit how to use this algorithm?
- What's _Expected LiFetime_ in the `guanrank_elf` function? I've read that's the area above KM curve but I don't know that's meaning exactly...
- Why guanrank score changes depending of using `guanrank` or `guanrank_elf`?

Thanks in advance for your help. Best regards.

**Comment by Hongyang449** [comment]
Hi @jgarces02 ,
Sorry about my late reply. 
`Is there any documentation or paper that helps me to understand a bit how to use this algorithm?`
This document can help you understand the GuanRank:
[GuanRank](https://pdfs.semanticscholar.org/3f15/fe4bf12ef58d3a9aa3264dc8d6125cba0e9c.pdf)
Some of our papers for your reference:
[myeloma](https://academic.oup.com/gigascience/article/8/12/giz153/5690077), [prostate cancer](https://www.sciencedirect.com/science/article/pii/S2589004219305504), [als](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005887)

`What's Expected LiFetime in the guanrank_elf function? I've read that's the area above KM curve but I don't know that's meaning exactly...`
This is a new version of GuanRank I created when I organized Yuanfang's original code. I call it Expected LiFetime (ELF), since it calculates the area under KM curve for each individual, which corresponds to the expectation of how long an individual will survival. This function is optional and you don't need it to calculate the original GuanRank.

`Why guanrank score changes depending of using guanrank or guanrank_elf?`
As I describe above, the core ideas of `guanrank` and `guanrank_elf` are slightly different - `guanrank` focuses on comparing each pair of individuals and ranked them, whereas `guanrank_elf` focuses on the mathematical expectation, or the area under KM curve of each individual. Therefore, the scores of these two are slightly different. Based on my experiences on 3 survival datasets, they have similar predictive performance.

Thank you,
Hongyang
